engine/crates/engine-v2/axum/src/middleware/telemetry.rs

```diff
@@ -174,19 +174,6 @@ where
                         latency,
                     );
 
-                    match gql_status {
```

Comment:
Removing it and replacing it with something more appropriate in the next PR only exposing the total count of errors and distinct error codes